.github/workflows/test-hyperlinks.md

```diff
@@ -335,7 +335,7 @@ Internal links are those that point to another file in the same repository or a
 ```
 
 ### Mid-page Anchor Links
-There is a special kind of links where the generated link points to a section header like this: [Anchor Links](#mid-page-anchor-links). 
+There is a special kind of links where the generated link points to a section header like this: [Anchor Links](#mind-page-anchor-links).
```

Comment:
There are special types of links where the generated link points to a section header in a page like this: ` [Anchor Links](#mid-page-anchor-links)`.

Reply:
Yes, I know. This bad link is only temporary there to show how the Hyperlink will list the broken link when this changes will be merged. As soon as the report format will be accepted I will restore this link.

Reply:
oh, ok, but there is a grammar issue, as well:
There is a special kind of links ...
would be better as:
There are special types of links where the generated link points to a section header in a page like this: